app/graphql/mutations/test_result/delete.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Mutations
+  module TestResult
+    # Mutation to delete a TestResult
+    class Delete < BaseMutation
+      include Fields
+
+      graphql_name 'deleteTestResults'
+
+      argument :profile_id, ID, required: true
+      field :profile, Types::Profile, null: false
+      collection_field :test_results, Types::TestResult
+
+      def resolve(args = {})
+        profile = scoped_profiles.find(args[:profile_id])
+        test_results = authorized_test_results(args)
+        test_results.map(&:destroy)
+
+        { profile: profile, test_results: ::RecordLoader.for(::TestResult).load_many(test_results)
```

Comment:
@dLobatog do you happen to know how I can return this array of test results? This doesn't do it :confused:

Reply:
It works for me, maybe you're querying it wrong? If you provide a Relay style resolver for that collection field, you need to query it in Relay style too:

```
mutation deleteTestResults($input: deleteTestResultsInput!) {
  deleteTestResults(input: $input) {
    profile {
      id
    }
    testResults {
      edges {
        node {
          id
        }
      }
    }
  }
}
```